app.js

```diff
@@ -67,32 +70,52 @@ for (var i = 0; i < stores.length; i++) {
   rowEl.appendChild(dailyTotalEl);
 }
 
+for (var i = 0; i < storeHours.length; i++) {
+  var hourAllStoreTotals = 0;
+  //var rowEl = document.createElement('tr');
+  //tableEl.appendChild(rowEl);
+  //var nameEl = document.createElement('th');
+  //nameEl.textContent = currentStore.name;
+  //rowEl.appendChild(nameEl);
+  for (var j = 0; j < stores.length; j++) {
+    var currentStore = stores[j];
+    hourAllStoreTotals += currentStore.hoursOpen[i];
+    //var storeHourEl = document.createElement('td');
+    //storeHourEl.textContent = currentStore.hoursOpen[j];
+    //rowEl.appendChild(storeHourEl);
+  }
+  storeHoursTotal.push(hourAllStoreTotals);
+}
+console.log(storeHoursTotal);
 document.body.appendChild(tableEl);
 
 console.log('————————— lab notes - event listeners—————————');
 
 //ask the Dom for the form
 var storeFormEl = document.getElementById('new-store-form');
+//adding a submit action not connecting to html - storeFormEl^^ is an html node - holding an instance of an html element node -
+storeFormEl.addEventListener('submit', handleSubmit); //adding false(force to bubble)/true(force to capture) as third parameter it swtiches them. stopPropagation in function negates needing this. addEventListener can be for any html element node
 
-storeFormEl.addEventListener('submit', handleSubmit); //adding false(force to bubble)/true(force to capture) as third parameter it swtiches them. stopPropagation in function negates needing this.
-
-function handleSubmit(event){
+function handleSubmit(event){ //using this function to call addEventListener - it is waiting for the submit event to happen - we never actually call it - we hand it off to the addEventListener - passing the function into addEventListener
```

Comment:
Your comments are great! 

We never need to call it because javascript will call it for us whenever it hears, in this case, a 'submit' event.